examples/user-guide/08-regridding.py

```diff
@@ -0,0 +1,456 @@
+# object. There are 2 caveats:
+# 
+# 1) the synchronization between idomain and the package domains is done on the
+#    model-level. If we have a simulation containing both a flow model and a
+#    transport model then idomain for flow is determined independent from that for
+#    transport. These models may therefore end up using different domains (this
+#    may lead to undesired results, so a manual synchronization may be necessary
+#    between the flow regridding and the transport regridding) This manual
+#    synchronization can be done using the "mask_all_packages" function- this
+#    function removes input from all packages that are marked as inactive or VPT
+#    in the idomain passed to this method.
+# 2) The idomain/packages synchronization step is carried out when regridding a
+#    model, and when regridding a model it will use default methods for all the
+#    packages. So if you have regridded some packages yourself with non-default
+#    methods, then these are not taken into account during this synchonization
+#    step. 
+# 
+# Regridding using default methods
+# ================================
+# 
+# The regrid_like function is available on packages, models and simulations.
+# When the default methods are acceptable, regridding the whole simulation is the most convenient 
+# from a user-perspective. 
+# 
+# 
+# Regridding using non-default methods
+# ====================================
+# 
+# When non-default methods are used for one or more packages, these should be
+# regridded separately. In that case, the most convenient approach is likely:
+# 
+# - pop the packages that should use non-default methods from the source simulation (the
+#   popping is optional, and is only recommended for packages whose presence is not
+#   mandatory for validation.) 
+# - regrid the source simulation: this takes care of all the packages that should use default methods. 
+# - regrid the package(s) where you want to use non-standard rergridding methods indivudually starting from the
+#   packages in the source simulation 
+# - insert the custom-regridded packages to the
+#   regridded simulation (or replace the package regridded with default methods with
+#   the one you just regridded with non-default methods if it was not popped) 
+# 
+# In code, consider an example where we want to regrid the recharge package using non default methods
+# then we would do the following. First we'll load some example simulation.
+# There is a separate example contained in :doc:`/examples/mf6/hondsrug`
+# that you should look at if you are interested in the model building
+import imod
+
+tmpdir = imod.util.temporary_directory()
+
+original_simulation = imod.data.hondsrug_simulation(tmpdir / "hondsrug_saved")
+
+# %%
+# The model was written before the xt3d_option and rhs_option arguments were
+# added to iMOD Python. Set missing options to False.
+original_simulation["GWF"]["npf"].set_xt3d_option(is_xt3d_used=False, is_rhs=False)
```

Comment:
can't we update the packages in imod.data.hondsrug_simulation with the set_xt3d_options? Then we don't have to do it here anymore, making this user-guide more consise.

Reply:
Good point, fixed by setting the xt3d option in the sample_data.py immediately